src/lib/stellar/horizonQueries.js

```diff
@@ -225,6 +225,6 @@ export async function findStrictReceivePaths({
     if (response.records.length > 0) {
         return response.records
     } else {
-        throw error(400, { message: 'no strict receive paths available' })
+        throw new Error('no strict receive paths available')
```

Comment:
We actually _do_ want to use the `error()` function here. It's a [SvelteKit-specific thing](https://kit.svelte.dev/docs/errors#expected-errors), but we're using it throughout this file to throw expected errors.